alfasim_sdk/hook_specs.py

```diff
@@ -1333,6 +1333,52 @@ def calculate_entrained_liquid_fraction(
     """
 
 
+def update_plugins_internal_deposit_layer(ctx: "void*", deposition_layer_thickness: "void*",
```

Comment:
I suggest 

```suggestion
def update_internal_deposit_layer(ctx: "void*", deposition_layer_thickness: "void*",
```

The fact that we are already in a plugin is enough. I see we have `update_plugins_secondary_variables` and you probably followed suit, but I think my suggestion is better, just not worth updating `update_plugins_secondary_variables` too.

Reply:
Done it